common/culture/cultures/wc_avianic.txt

```diff
@@ -6,14 +6,20 @@
 	language = language_darnassian
 	martial_custom = martial_custom_female_only
 	traditions = {
+		tradition_hill_dwellers
+		tradition_frugal_armorsmiths
+		tradition_adaptive_skirmishing
+		tradition_practiced_pirates
+		tradition_hunters
+		tradition_nubian_warrior_queens
```

Comment:
The following are cultures already assigned to either Warrior Queens or Matriarchal. I'll qualify them as a baseline for Warrior Queens.
Warrior Queens: Harpy, Naga, Shivarra, Hyldnir, Blood Trolls. 
Matriarchal: Mantid

As for those cultures not assigned either, I propose the following:
- Sayaadi
  - Though they are genderfluid beings and prefer not to directly confront their enemies, they are adaptable beings.
- Durotaric
  - Considering Thrall's progressive reforms such as emancipating peons from slavery.
- Tauren
  -  Magatha Grimtotem?
- Darnassian
  - The Long Vigil and Sisterhood of Elune.
- Zulite heritage
  - Female leadership may not be the default but Trolls don't seem opposed to it if the leader demonstrates their competence.

Do you have other suggestions?

Reply:
> Darnassian

Probably that's the only culture I would give Warrior Queens from the suggested list. I would like it strict.
Maybe even the night elvens should have it from the begging since only females serve in the army.